src/modules/back-to-top/back-to-top.component.html

```diff
@@ -0,0 +1,3 @@
+<div #backToTop class="back-to-top" (click)="goToTop()" [ngStyle]="{'background-color': backgroundColor, 'color': color}">
```

Comment:
The word `scroll` better represents the action taken here ("go" could imply a URL change).